spyder/widgets/sourcecode/base.py

```diff
@@ -1379,20 +1379,17 @@ def append_text_to_shell(self, text, error, prompt):
             text = text[index+1:]
             self.clear()
         if error:
-            is_traceback = False
             for text in text.splitlines(True):
                 if text.startswith('  File') \
                 and not text.startswith('  File "<'):
-                    is_traceback = True

```

Comment:
We need to keep this to activate the dialog only when text printed in the internal console is *really* a traceback. Else the error dialog is shown for things like this

![seleccion_001](https://user-images.githubusercontent.com/365293/27944279-7be884d4-62ab-11e7-99a5-4c91b9d15601.png)

which is no real error, just a warning printed through stderr :-)